spacy_llm/tasks/ner/task.py

```diff
@@ -47,6 +51,8 @@ def __init__(
         single_match (bool): If False, allow one substring to match multiple times in
             the text. If True, returns the first hit.
         scorer (Scorer): Scorer function.
+        description (str): todo
```

Comment:
TODO

Reply:
Done in https://github.com/explosion/spacy-llm/pull/180/commits/3a31530bfd257cd77028a964e9b1e421676c0abd.